Makefile

```diff
@@ -71,4 +71,10 @@ $(GOBIN)/auth0-cli-config-generator:
 
 integration: $(GOBIN)/auth0-cli-config-generator $(GOBIN)/commander
 	auth0-cli-config-generator && commander test commander.yaml
+	$(MAKE) integration-cleanup
```

Comment:
perhaps a nitpick here, but we could replace recursive use of `make` with:

```
run-integration:
    auth0-cli-config-generator && commander test commander.yaml
.PHONY: run-integration

integration: $(GOBIN)/auth0-cli-config-generator $(GOBIN)/commander run-integration integration-cleanup
.PHONY: integration
```

Reply:
Updated